docs/team/joel-sung.md

```diff
@@ -42,10 +42,10 @@ Given below are my contributions to the project.
   * Maintained the issue tracker: most issues were created and assigned by me
   * In charge of the release of v1.3
 * Reviewing/mentoring contributions:
-  * Reviewed many PRs and gave my feedback. These are some of the more significant ones:
-    * https://github.com/AY2122S1-CS2103T-T11-1/tp/pull/97
-    * https://github.com/AY2122S1-CS2103T-T11-1/tp/pull/96
-    * https://github.com/AY2122S1-CS2103T-T11-1/tp/pull/59
-    * https://github.com/AY2122S1-CS2103T-T11-1/tp/pull/94
+  * [Reviewed many PRs and gave my feedback.](https://github.com/AY2122S1-CS2103T-T11-1/tp/pulls?q=is%3Apr+is%3Aclosed+reviewed-by%3A%40me)
```

Comment:
The link here shows the PRs reviewed by the person who is logged in with their Github account. So when I accessed the link it showed me the PRs that I reviewed.
```suggestion
  * [Reviewed many PRs and gave my feedback.](https://github.com/AY2122S1-CS2103T-T11-1/tp/pulls?q=is%3Apr+is%3Aclosed+reviewed-by%3Ajoel-sung)
```
You can try this link. Instead of filtering by `is:pr is:closed reviewed-by:@me` you can do `is:pr is:closed reviewed-by:joel-sung`